### PR TITLE
Containerize VBMC and add redfish support

### PR DESCRIPTION
**Pull request by maelk**
- run vbmc as a container
- run sushy-tools for redfish interface as a container

### CODE REVIEW

**Review by jan-est — DISMISSED** [review]

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_ubuntu/1/

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/1/

**Review by nordixinfra — COMMENTED** [review]
:broken_heart: Test Failure : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/1/

**Review by nordixinfra — COMMENTED** [review]
:green_heart: Test Success : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_ubuntu/1/

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/2/

**Review by nordixinfra — COMMENTED** [review]
:broken_heart: Test Failure : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/2/

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/3/

**Review by nordixinfra — COMMENTED** [review]
:broken_heart: Test Failure : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/3/

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/4/

**Review by nordixinfra — COMMENTED** [review]
:broken_heart: Test Failure : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/4/

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/5/

**Review by nordixinfra — COMMENTED** [review]
:broken_heart: Test Failure : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/5/

**Review by nordixinfra — COMMENTED** [review]
:revolving_hearts: Test pending : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/6/

**Review by nordixinfra — COMMENTED** [review]
:broken_heart: Test Failure : https://jenkins.nordix.org/job/airship_nordix_metal3_dev_env_integration_test_centos/6/

File: resources/sushy-tools/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM registry.hub.docker.com/library/python:3.7
+
+RUN apt update && \
```
Comment (Xenwar) [inline resources/sushy-tools/Dockerfile]: same

**Pull request merged**